src/fractional_coloring.jl

```diff
@@ -0,0 +1,56 @@
+function indvec(s, n)
+    x = zeros(n)
+    x[s] .= 1
+    return x
+end
+
+"""
+    fractional_chromatic_number(g; optimizer)
+
+Compute the fractional chromatic number of a graph.  Gives the same result as
+fractional_clique_number, though one function may run faster than the other.
+Beware: this can run very slowly for graphs of any substantial size.
+
+# Keyword arguments
+
+- `optimizer`: JuMP-compatible solver (default is `GLPK.Optimizer`)
+"""
+function fractional_chromatic_number(
+    g::AbstractGraph{T}, optimizer=GLPK.Optimizer
```

Comment:
Does it work with directed graphs?

Reply:
No, it does not (and as far as I know is not defined for directed graphs).  I've added a check.